### PR TITLE
refactor: docs, player

### PR DESCRIPTION
**Pull request by looshch**
Add Work Log. Extend the logic for controlling Pudge. Move Prettier
config to the root. Update TypeScript’s locked dependency in the root.

Refactor the app into class-based components. Add favicons and the logo.

### CODE REVIEW

File: package-lock.json
Comment (looshch) [inline package-lock.json:1]: intended

File: readme.md [hunk 1]
```diff
@@ -201,4 +204,34 @@
    - player has thrown a hook;
    - player has cancelled throwing of a hook.
 
+## Work Log
```
Comment (looshch) [inline readme.md:207]: let’s leave short logs here in the end of each day we are working on it, will be interesting to check them later

File: ui/src/components/app/app.ts [hunk 2]
```diff
@@ -0,0 +1,10 @@
+import { Application, Renderer } from 'pixi.js'
+
+export type App = Application<Renderer>
+
+export const initAndGetApp = async (): Promise<App> => {
```
Comment (looshch) [inline ui/src/components/app/app.ts:5]: using classes only if it makes sense

File: ui/tsconfig.json [hunk 3]
```diff
@@ -5,15 +5,17 @@
     "useDefineForClassFields": true,
     "module": "ESNext",
     "lib": ["ES2020", "DOM", "DOM.Iterable"],
-    "moduleResolution": "bundler",
```
Comment (looshch) [inline ui/tsconfig.json:8]: these options were tripping over `tsc` command

**Review by brempavel — APPROVED** [review]

**Pull request merged**